test/Model/ReviewedPreprintTest.php

```diff
@@ -0,0 +1,221 @@
+<?php
+
+namespace test\eLife\ApiSdk\Model;
+
+use eLife\ApiSdk\Collection\ArraySequence;
+use eLife\ApiSdk\Collection\EmptySequence;
+use eLife\ApiSdk\Collection\Sequence;
+use eLife\ApiSdk\Model\File;
+use eLife\ApiSdk\Model\Image;
+use eLife\ApiSdk\Model\ReviewedPreprint;
+use PHPUnit_Framework_TestCase;
+
+final class ReviewedPreprintTest extends PHPUnit_Framework_TestCase
+{
+    /**
+     *         string $id,
+    string $title,
+    string $status,
+    string $stage,
+    string $indexContent = null,
+    string $doi = null,
+    string $authorLine = null,
+    string $titlePrefix = null,
+    DateTimeImmutable $published = null,
+    DateTimeImmutable $reviewedDate = null,
+    DateTimeImmutable $statusDate = null,
+    int $volume = null,
+    string $elocationId = null,
+    string $pdf = null,
+    Sequence $subjects = null,
+    Sequence $curationLabels = null,
+    Image $image = null
+     * @test
+     */
+    private $reviewedPreprint;
+    private $emptyReviewedPreprint;
+
+    public function setUp()
+    {
+        $this->reviewedPreprint = new ReviewedPreprint(
+            'id',
+            'title',
+            'status',
+            'stage',
+            'indexContent',
+            'doi',
+            'authorLine',
+            'titlePrefix',
+            new \DateTimeImmutable('2022-08-31'),
```

Comment:
Declare `DateTimeImmutable` in use statement and use ISO8601 standard

See: https://github.com/elifesciences/api-raml/blob/develop/src/misc/date-time.v1.yaml

or examples in other similar tests